Formula/s/swig.rb

```diff
@@ -5,6 +5,11 @@ class Swig < Formula
   sha256 "fa045354e2d048b2cddc69579e4256245d4676894858fcf0bab2290ecf59b7d8"
   license "GPL-3.0-or-later"
 
+  livecheck do
+    url :stable
```

Comment:
```suggestion
    url "https://sourceforge.net/projects/swig/rss?path=/swig"
```

The upstream project contains multiple directories (`swigwin`, `swig`, `OldFiles`, `macswig`) but we're using tarballs from the `swig` directory, so we should check the RSS feed for that directory. This helps to ensure that the files we're interested in aren't pushed out of the RSS feed by files from other directories.